src/static/pages/main/views/layout.jade

```diff
@@ -2,10 +2,13 @@ doctype html
 html
   head
     title= title
-    link(rel="stylesheet", href="/css/style.css")
+    link(rel="stylesheet", href="/css/bootstrap.css")
+    link(rel="stylesheet", href="/css/main.css")
     script(type="text/javascript", src="/socket.io/socket.io.js")
   body
     header
         include ./header.jade
-    article
+    .container
         block content
+
+    script(type="text/javascript", src="/js/dependencies.js")
```

Comment:
I'd move the whole dependencies.js file, since any dependency would be used in files imported in *content*